miden-lib/asm/miden/kernel_invocation.masm

```diff
@@ -0,0 +1,375 @@
+# KERNEL PROCEDURE OFFSETS
+# =================================================================================================
+
+# OFFSET CONSTANTS
+# -------------------------------------------------------------------------------------------------
+
+# The offsets at which hash of the corresponding procedure is stored.
+# const.EXEC_KERNEL_PROC_OFFSET=0
```

Comment:
I'm not sure `EXEC_KERNEL_PROC_OFFSET` needs to be here. We should be able to start with with offset 0 for the actual procedures invoked via offset.

Reply:
Yep, I agree, but it is just a coincidence that `EXEC_KERNEL_PROC_OFFSET` has the 0 offset, in other iterations it had some other offset which was just skipped, and offset 0 corresponded to some actual procedure. 
I decided to leave this proc commented to make it visual why did we skip some offset in the random place in the middle of the offset list.

Reply:
> But after I implement the hashes array we can get rid of the `EXEC_KERNEL_PROC_OFFSET`, since I can place it at the last position.

I would actually say that `EXEC_KERNEL_PROC` should not be in that array at all.